bugwarrior/services/bz.py

```diff
@@ -65,6 +73,14 @@ def to_taskwarrior(self):
         if self.extra.get('needinfo_since', None) is not None:
             task[self.NEEDINFO] = self.extra.get('needinfo_since')
 
+        # iff field is defined, use it, converting None to empty string.
+        if 'assigned_on' in self.extra:
+            if self.extra['assigned_on']:
+                task[self.ASSIGNED_ON] = self.extra.get('assigned_on')
+            else:
+                task[self.ASSIGNED_ON] = ''
```

Comment:
Seems more straightforward to me: `task[self.ASSIGNED_ON] = self.extra.get('assigned_on') or ''`